code/src/reducers/todos.js

```diff
@@ -0,0 +1,62 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+
+
+const initialState = {
+    items: [
+        {id: 0, text:"call queen mother", isCompleted: false},
+        {id: 1, text:"make the app work", isCompleted: false},
+        {id: 2, text:"groceries and bolaget", isCompleted: false}
+    ]
+};
+
+//slice is always composed of input, initialstate and reducers
+export const todos = createSlice({
+    name: "todos",
+    initialState,
+    reducers: {
+        addItem: (store, action) => {
+        
+            const newItem = { 
+                // spread the current store and return maximum value of array +1 
+                id: Math.max(...store.items.map(item => item.id)) + 1,
```

Comment:
There's a good library for creating unique id's -> https://www.npmjs.com/package/uuid

Reply:
Thank you!

Reply:
Code is now up to date and mentioned issues are resolved 🧷